sdk/core/azure-core-experimental-perf/src/main/java/com/azure/core/perf/ARMChallengeAuthenticationPolicyTest.java

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.perf;
+
+import com.azure.core.credential.AccessToken;
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.credential.TokenRequestContext;
+import com.azure.core.experimental.http.policy.ARMChallengeAuthenticationPolicy;
+import com.azure.core.experimental.http.policy.BearerTokenAuthenticationChallengePolicy;
+import com.azure.core.experimental.implementation.AzureEnvironment;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+import com.azure.core.http.HttpRequest;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.rest.RestProxy;
+import com.azure.core.perf.core.MockHttpClient;
+import com.azure.core.perf.core.MyRestProxyService;
+import com.azure.core.perf.models.MockHttpResponse;
+import com.azure.core.util.CoreUtils;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.perf.test.core.PerfStressOptions;
+import com.azure.perf.test.core.PerfStressTest;
+import reactor.core.publisher.Mono;
+
+import java.time.OffsetDateTime;
+import java.util.HashMap;
+
+/**
+ * The Performance Test class for Bearer Token Authentication Challenge Policy.
+ */
+public class ARMChallengeAuthenticationPolicyTest extends PerfStressTest<PerfStressOptions> {
```

Comment:
Why is this named ARM? This isn't related to the management plane right?

Reply:
Yes, this policy is for ARM SDK.
We will work with ARM folks and move it there later on.